frappe/utils/background_jobs.py

```diff
@@ -18,12 +18,17 @@
 from frappe import _
 from frappe.utils import cstr
 
+site_config = frappe.get_site_config()
```

Comment:
This line can't be in globals. `frappe.utils.background_jobs.site_config` will fetch the wrong site config based on which site is initialized first.

Reply:
Hello @gavindsouza 

Thanks for the feedback!.
Okay. I'm going to remove the site config.